README.md

```diff
@@ -49,3 +49,14 @@ Details on how to configure such a (TP-Link TL-WR802N) router can be found [here
 - A schematic of the LED strip can be found [here](docs/images/raspberry_led_platine_v2.pdf)
 
 >Instructions for Display assembly are available [here](docs/display/display-assembly-instructions.md)
+
+
+## Running 64-bit Ubuntu
+
+Ubuntu has a 64-bit ARM version with built-in cloud-init support. The `akka-pi-os-dhcp-64.yml` cloud-init file has the required modifications to flash a card for that version of the OS on ARM.
```

Comment:
should we mention "ubuntu" in file name?

Reply:
Don't think so: the yaml file is only linked to an os instance that has support for cloud-init.

Reply:
But you may be right that the way I formulate this links it too tight to Ubuntu. I'll reformulate it a bit.